Cheatsheet.md

```diff
@@ -0,0 +1,504 @@
+# Cheatsheet
+## Commonly used functions on [Array](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Arrays)
+### [**1. map**](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)

```

Comment:
please follow proper md formatting.  These ### headers should not be bolded as well, please remove the ** from all of them.

Reply:
ok.

Reply:
> ok.

done